assets/js/modules/adsense/components/module/ModuleOverviewWidget/ModuleOverviewWidget.stories.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * ModuleOverviewWidget component stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { WithTestRegistry } from '../../../../../../../tests/js/utils';
+import * as fixtures from '../../../datastore/__fixtures__';
+import { STORE_NAME } from '../../../datastore/constants';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { CORE_WIDGETS, WIDGET_WIDTHS, WIDGET_AREA_STYLES } from '../../../../../googlesitekit/widgets/datastore/constants';
+import WidgetAreaRenderer from '../../../../../googlesitekit/widgets/components/WidgetAreaRenderer';
+import ModuleOverviewWidget from './index';
+
+export const Performance = () => {
```

Comment:
This would only export the single story; we should still have stories for the other states like we do now with the generate-widget-based stories utility.

Per @felixarntz 's https://github.com/google/site-kit-wp/pull/3023#issuecomment-809621278, it looks like I was wrong about going full SB6 here just yet, so feel free to use the old approach for now until we've fully fleshed out how the new approach will work going forward, including these multi-variant cases 👍